atest/README.rst

```diff
@@ -105,3 +105,9 @@ Tests also require robotstatuschecker:
     pip install robotstatuschecker
  
 Tests are ran using Bash script `python atest/run.py`. The script prints help when ran without parameters.
+
+In order to run the tests with IPv6, the ``::1`` must be used as host variable when running ``atest/run.py`` script.
+
+Example:
+::
```

Comment:
The above two lines can be combined to just `Example::` (or perhaps the two lines could be remove and the previous sentence just end with `... when running atest/run.py script::`. For details about this reStructuredText feature see:
http://docutils.sourceforge.net/docs/user/rst/quickref.html#literal-blocks